2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:
  */
 - (void)doConnect
 {
-    OTError *error;
+    OTError *error = nil;
```

Comment:
Internally, we don't use ARC because we have a lot of asynchronous memory management with internal threads and it's been more reliable to manage things explicitly. There's no reason the samples should or should not have the feature enabled, however.